src/server/debug.rs

```diff
@@ -624,6 +662,131 @@ impl<ER: RaftEngine> Debugger<ER> {
         Ok(())
     }
 
+    pub fn remove_regions(&self, region_ids: Vec<u64>) -> Result<()> {
```

Comment:
Seems using `tikv-ctl tombstone` is enough?

Reply:
Will TiKV delete the data if a region is tombstoned?

Reply:
yes

Reply:
fixed